lib/linalg/svd/RandomizedSVD.cpp

```diff
@@ -104,7 +104,7 @@ RandomizedSVD::computeSVD()
         rand_mat = new Matrix(snapshot_matrix->numColumns(), d_subspace_dim, false);
         for (int i = 0; i < snapshot_matrix->numColumns(); i++) {
             for (int j = 0; j < d_subspace_dim; j++) {
-                rand_mat->item(i, j) = 1;
+                rand_mat->item(i, j) = (i == j);
```

Comment:
need to change the text above, e.g., 

If debug mode is turned on, just set rand_mat as a matrix of 1s if i==j, zero otherwise.

Reply:
Yes, the comment can just say it is set to the identity matrix in debug mode.

Reply:
It's a "truncated identity matrix" to be more accurate.

Reply:
changed the comments.